lib/VisualGenerator.js

```diff
@@ -142,6 +239,7 @@ class VisualGenerator {
                     }
 
                     copyVisualTemplate(visualPath, buildOptions.template);
+                    generateSettings(visualPath);
```

Comment:
We should think of how to do an 'update settings' option for the user. Meaning that once the user goes and updates his capabilities.json he can then ask to just update his settings.ts file.
I think we can do this in a separate PR \ iteration.